src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -2425,6 +2425,7 @@ public function get_qty_pending( $ticket_id, $refresh = false ) {
 	public function attendee_decreases_inventory( array $attendee ) {
 		$order_status = Tribe__Utils__Array::get( $attendee, 'order_status', 'undefined' );
 		$order_id = Tribe__Utils__Array::get( $attendee, 'order_id', false );
+		$attendee_id = Tribe__Utils__Array::get( $attendee, 'attendee_id', false );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 2 spaces but found 1 space

<sub>hash: 47e442ec038f91ff0e53b9dfb771e1d4</sub>